README.md

```diff
@@ -5,9 +5,10 @@ Customers are expected to have high number of GCP projects. So this module can h
 That allows you to create GCP credential config in Google Cloud Platform projects which will be used to get GCP data from AWS environment.
 
 This module will create below resources:-
- * It creates host folder, host project , service account, work pool identity & add cloud provider to it.
- * It will update IAM permission of each selected project to allow access by the new Service Account.
- * It will attach below policies to service account of host projects and other integration projects.
+ * It creates host folder, host project. , service account, work pool identity & add identity provider to it.
+ * It creates service account, work pool identity & identity provider under host project.
```

Comment:
It seems duplicate of `It creates host folder, host project. , service account, work pool identity & add identity provider to it.`

Reply:
Got it. Will fix